index.html

```diff
@@ -223,6 +223,38 @@ <h4 class="uppercase mb16">Community Connect</h4>
 	</div>
 </section>
 
+<!-- Social Embed -->
+<section class="image-bg overlay parallax">
+    <div class="background-image-holder">
+        <img alt="Background Image" class="background-image" src="img/cover11.jpg" />
```

Comment:
@YashKumarVerma, in your PR you have used `cover11.jpg` here as the background image for social embed block but I cannot find this image in the `/img/` directory and hence the background image failed to load. Could you please reply back with the image intended to be `cover11.jpg` as I'm fixing this issue as a part of #678

Reply:
I've found the image after looking at the repository at the point of time when you added the PR. I'll add that to the repository. Thanks anyway :)